sdk/storage/Azure.Storage.Blobs.DataMovement/tests/Azure.Storage.Blobs.DataMovement.Tests.csproj

```diff
@@ -7,13 +7,17 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Security.KeyVault.Keys" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\src\Azure.Storage.Blobs.DataMovement.csproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common.DataMovement\tests\Azure.Storage.Common.DataMovement.Tests.csproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Blobs\tests\Azure.Storage.Blobs.Tests.csproj" />
```

Comment:
I had to add Constants to get BlobTestBase methods working, and the Tests project in as well. But, this causes a conflict in BlobScannerTests.cs at line 47 with .ToListAsync(), because both Blobs.Tests and DM.Tests point to Common, and somehow VS says there's ambiguity between the two references to the same method. I removed the DM.Tests package for now, since it isn't being used yet and I wanted to test the scanner. Any ideas on how to actually deal with that issue? I tried rebuilding, but that didn't seem to do it.

Forgot to remove Security reference from trying to import everything similar to Blobs.Batch.Tests to avoid removing DM.Tests package (didn't work); gonna take it out.